demo/multicluster-fault-injection.sh

```diff
@@ -43,9 +43,6 @@ spec:
         openservicemesh.io/sidecar-injection: "false"
     spec:
       serviceAccountName: "$BOOKSTORE_SVC"
-      nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
```

Comment:
Same here to keep os but remove arch.

Reply:
Why is it needed here? Just curious.

Reply:
curl doesn't have Windows images but supports several architectures for linux.

Reply:
Ah, I only had to look a measly two lines down. Thank you so much :) That makes sense.